js/src/frameworks/langgraph.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, it, expect, beforeAll } from "@jest/globals";
+import { z } from "zod";
+import { getTestConfig } from "../../config/getTestConfig";
+import { LanggraphToolSet } from "./langgraph";
+
+
+// Langgraph extens langchain class, all the properties are same
+describe("Apps class tests", () => {
```

Comment:
The test description 'Apps class tests' is incorrect. It should be 'LanggraphToolSet tests' since we're testing the LanggraphToolSet class.